src/app/core/entity-components/entity-list/entity-list.module.ts

```diff
@@ -48,6 +49,7 @@ import { MatDialogModule } from "@angular/material/dialog";
     ExtendedModule,
     MatButtonToggleModule,
     MatTableModule,
+    MatProgressSpinnerModule,
```

Comment:
```suggestion
```
This should not be needed here as the loading indicator is in the entity subrecord